### DISCUSSION TITLE
On windows 11 on a TPM Enabled laptop (Dell 7420) after i restart OS from hibernation error on TPM popup

### DISCUSSION THREAD

**Discussion by MicheleNicosia** [keepassxreboot/keepassxc#10738] [Q&A]
Hello All,
  recently my laptop got a motherboard change due failure, from that point i noted after i stop on HIBERNATE, and restart the OS, KeypassXC ask for the password again instead fo using the Windows HELLO (face,pin,fingerprint always work).

So it happens so: 

1. Keypassxc working well
2. Hybernation 
3. Coming back live
4. KeypassXC use Windows Hello for the DB opened
5. Error Popup on TPM 2.0 usage 
6. I need to insert the PASSWORD again for the already opened DB
7. KeypassXC use Windows Hello for the DB opened
8. Works fine

Is that correct?
How do avoid, as i recall it was workign without the points 5 and 6 before the MB change.

Thank you

**Comment by droidmonkey** [comment]
I have seen that before as well, but haven't since using 2.7.8. The error thrown is from windows so nothing much we can do about it. I translate it to the key is no longer valid or available from the TPM to do the decryption.
